jarbas/frontend/elm/Documents/Inputs/View.elm

```diff
@@ -10,6 +10,42 @@ import Material.Textfield as Textfield
 import Material.Typography as Typography
 import Documents.Inputs.Model exposing (Model, Field, model)
 import Documents.Inputs.Update exposing (Msg(..), update)
+import Internationalization exposing (TranslationId(..), translate)
+import Regex
+
+
+{-| Matches a date un the YYYY-MM-DD format
+
+    >>> matchDate "1970-01-01"
+    True
+
+    >>> matchDate "foo-42-01"
+    False
+
+-}
+matchDate : String -> Bool
+matchDate value =
+    if String.isEmpty value then
+        True
+    else
+        let
+            regex : Regex.Regex
+            regex =
+                Regex.regex
+                    "[\\d]{4}-[\\d]{2}-[\\d]{2}"
```

Comment:
Actually you have to escape the slashes, so you suggestion would transform in `\\d{4}\\-\\d{2}\\-\\d{2}` which I found kind of confusing to read — so I opted for the brackets. But as far as I can remember both worked when I was coding it.